config/localTemplate.go

```diff
@@ -76,7 +77,52 @@ type Local struct {
 	BaseLoggerDebugLevel uint32 `version[0]:"1" version[1]:"4"`
 	// if this is 0, do not produce agreement.cadaver
 	CadaverSizeTarget uint64 `version[0]:"1073741824" version[24]:"0"`
-	CadaverDirectory  string `version[27]:""`
+	// if this is not set, MakeService will attempt to use ColdDataDir instead
+	CadaverDirectory string `version[27]:""`
+
+	// HotDataDir is an optional directory to store data that is frequently accessed by the node.
+	// For isolation, the node will create a subdirectory in this location, named by the genesis-id of the network.
+	// If not specified, the node will use the runtime supplied datadir to store this data.
+	// Individual resources may have their own override specified, which would override this setting for that resource.
+	// Setting HotDataDir to a dedicated high performance disk allows for basic disc tuning.
+	HotDataDir string `version[29]:""`
```

Comment:
29 -> 31 here and below (30 is going to be cut today or Monday), plus a new `config-v31.json` with `make generate`

Reply:
ah yep. Have config 30 in here but not 31 yet